pages/vi/vi-nation.md

```diff
@@ -2,9 +2,9 @@
 
 ## Objectives
 
-* Learn to sync your Community with the Nation
-* Use the Nation interface to check whether your Community successfully joined the Nation
-* Update your Community Planet to the latest version
+- Learn to sync your Community with the Nation
```

Comment:
Are these intentional?

Reply:
@smitshrestha101 This change is not required as `*` also does generate list. So before making changes please also check if it is having any issue or not